frontend/packages/core-api-client/src/lib/types/client/Recipient.ts

```diff
@@ -0,0 +1,16 @@
+import { Record, RecordList } from '../model';
+import {DataOperation} from "./utils";
+import {
+  RecordLookup,
+  FormLookup,
+} from "./Record";
+
+export type Recipient = Record;
+export type RecipientDefinition = Omit<Recipient, 'id'>
+export type RecipientList = RecordList;
+
+export interface RecipientClientDefinition {
+  create: DataOperation<RecipientDefinition, Recipient>;
+  list: DataOperation<FormLookup , RecipientList>;
+  get: DataOperation<RecordLookup, Recipient>;
```

Comment:
I don't think a `DataOperation` is the correct type here. It works be accident as the methods have a single argument, but would fail with additional arguments.

Reply:
not if we stick to passing arguments as an object